pkg/agent/agent_windows.go

```diff
@@ -150,14 +150,14 @@ func (i *Initializer) prepareOVSBridge() error {
 	if err = util.SetAdapterMACAddress(brName, &uplinkNetConfig.MAC); err != nil {
 		return err
 	}
-	// Remove existing IP addresses to avoid a candidate error of "Instance MSFT_NetIPAddress already exists" when
-	// adding it on the adapter.
-	if err = util.RemoveIPv4AddrsFromAdapter(brName); err != nil {
```

Comment:
`RemoveIPv4AddrsFromAdapter` is dead code now, should it be removed?

Reply:
Sure, wii remove it in update.

Reply:
Done.